webapp/channels/src/components/about_build_modal/about_build_modal.tsx

```diff
@@ -182,6 +189,48 @@ export default class AboutBuildModal extends React.PureComponent<Props, State> {
 
         const mmversion: string | undefined = config.BuildNumber === 'dev' ? config.BuildNumber : config.Version;
 
+        let serverHostname;
+        if (!this.props.socketStatus.connected) {
+            serverHostname = (
+                <div>
+                    <FormattedMessage
+                        id='about.serverHostname'
+                        defaultMessage='Hostname:'
```

Comment:
Do we need the `:` inside the translated string? Mostly asking in case we could reuse the `Hostname` original string if it's used somewhere else . cc: @cwarnermm

Reply:
> _Do we need the `:` inside the translated string?_

I included the `:` to keep this consistent with other entries for this modal:

https://github.com/mattermost/mattermost/blob/d43f12b969d7e4a2ed10fb6e58c8dc69d2267161/webapp/channels/src/i18n/en.json#L12-L14

I'm happy to remove it if that makes sense, though :slightly_smiling_face: 

> _[...] we could reuse the Hostname original string if it's used somewhere else [...]_

I think if we wanted to use this string elsewhere, we probably want a different entry specific to the other use case. The text here may not always be `Hostname:`. For example, maybe we want to change it to `Cluster Node:` or add extra context like `Websocket Hostname:`. WDYT?

Reply:
I suspect these specific strings aren't reused elsewhere across the platform (and possibly because of the colons).

Reply:
Gotcha, thanks for the information.